CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 # Azure Kubernetes Service Changelog
+## Release 2023-05-14
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+ 
+* Docker container runtime for Windows nodepools has been retired as of May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
+* Each Kubernetes version is supported for 12 months. After 12 months, the minor version will shift to platform support only. Our new [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy) provides customers with Azure infrastructure support while the cluster is in an n-3 version (where n is the latest supported AKS GA minor version). Platform support does not include anything related to Kubernetes functionality and components, but provides customers with additional support beyond what was previously provided for unsupported versions.
+* The Docker Bridge CIDR field in the AKS API was made redundant during our change from Docker to containerD in Kubernetes version 1.19. Starting with the 2023-04-01 AKS API version, the Docker Bridge CIDR field will be removed. 
+* AKS is gradually rolling out a change that will rotate the token in the kubeconfig credentials. It shall not incur any impact since kubeconfig has the client certificate. Should you see any issue, retrieve the kubeconfig again with `az aks get-credentials`.
+
+### Release notes
+
+* Behavior Changes
+   * Customers can now upgrade AKS private clusters to [apiserver vnet integrated clusters](https://learn.microsoft.com/en-us/azure/aks/api-server-vnet-integration#limitations) in all public cloud regions.
+   * Unattended Upgrades are disabled on Mariner when running on a NVIDIA GPU enabled VM sizes.
+   * SecurityPatch OS Servicing channel not supported on Mariner when running on NVIDIA GPU enabled VM sizes.
+
+* Bug Fixes
+   * Now returning a clientError if the Public IP is not found, when creating agent pool with invalid nodePublicIPPrefixID.
+   * For Node Restriction enabled enabled clusters running window calico, we added a new role to grant windows containers permission to get secret from calico-system.
+   * Now returning a clientError when Stop Cluster fails with ScaleVMSSAgentPoolFailed when there is no LB on the cluster.
+
+* Component Updates
```

Comment:
ubuntu and mariner notes?

Reply:
releases are coming out later this week.  Since we are posting today. Will skip to next week.